Library/Homebrew/test/support/helper/env.rb

```diff
@@ -1,14 +1,20 @@
 module Test
   module Helper
     module Env
+      def copy_env
+        ENV.to_hash
+      end
+
+      def restore_env(env)
+        ENV.replace(env)
+      end
+
       def with_environment(partial_env)
```

Comment:
We could even get rid of this entirely now, since we are only using this five times, with only one variable each.